Doc/library/enum.rst

```diff
@@ -761,11 +761,11 @@ Data Types
 Supported ``__dunder__`` names
 """"""""""""""""""""""""""""""
 
-:attr:`__members__` is a read-only ordered mapping of ``member_name``:``member``
+:attr:`!EnumType.__members__` is a read-only ordered mapping of ``member_name``:``member``
 items.  It is only available on the class.
 
-:meth:`__new__`, if specified, must create and return the enum members; it is
-also a very good idea to set the member's :attr:`_value_` appropriately.  Once
+``__new__``, if specified, must create and return the enum members; it is
+also a very good idea to set the member's ``_value_`` appropriately.  Once
```

Comment:
I'm even more confused. On previous lines, generic `object` methods were being changed to italic, but now they're being changed to verbatim code literals? Instead, these should either be linked to the description of the generic method (again, probably better):

```suggestion
:meth`~object.__new__`, if specified, must create and return the enum members; it is
also a very good idea to set the member's :attr:`!_value_` appropriately.  Once
```

Or, if for whatever reason that is undesired, keep the existing appropriate formatting/markup and just prevent the resolution:

```suggestion
:meth`!__new__`, if specified, must create and return the enum members; it is
also a very good idea to set the member's :attr:`!_value_` appropriately.  Once
```

Reply:
Done!